inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/repository/DataProxyConfigRepository.java

```diff
@@ -99,6 +100,7 @@ public class DataProxyConfigRepository implements IRepository {
     public static final String KEY_DATA_NODE_NAME = "defaultDataNodeName";
     public static final String KEY_SORT_CONSUMER_GROUP = "defaultSortConsumerGroup";
     public static final String KEY_SINK_NAME = "defaultSinkName";
+    public static final String KEY_INLONG_COMPRESS_TYPE = "inlongCompressType";
```

Comment:
```suggestion
    public static final String KEY_COMPRESS_TYPE = "compressType";
```

Reply:
This name is used to distinguish it from other compress types, such as hive's compress type.

Reply:
All right.